fvm/fvm_test.go

```diff
@@ -2448,3 +2448,71 @@ func TestAttachments(t *testing.T) {
 		test(t, false)
 	})
 }
+
+func TestCapabilityControllers(t *testing.T) {
+	test := func(t *testing.T, capabilityControllersEnabled bool) {
+		newVMTest().
+			withBootstrapProcedureOptions().
+			withContextOptions(
+				fvm.WithReusableCadenceRuntimePool(
+					reusableRuntime.NewReusableCadenceRuntimePool(
+						1,
+						runtime.Config{
+							CapabilityControllersEnabled: capabilityControllersEnabled,
+						},
+					),
+				),
+			).
+			run(func(
+				t *testing.T,
+				vm fvm.VM,
+				chain flow.Chain,
+				ctx fvm.Context,
+				snapshotTree snapshot.SnapshotTree,
+			) {
+				txBody := flow.NewTransactionBody().
+					SetScript([]byte(`
+						transaction {
+                          prepare(signer: AuthAccount) {
```

Comment:
The formatting on the cadence code is a bit weird, probably a mix of tabs and spaces.

Reply:
Hopefull fixed in 6af91b03b2bbd078cf8a288b24fa86149890e959